src/base/bios/x86/bios.s

```diff
@@ -706,7 +706,8 @@ LFN_42_A6_HELPER_COMMON:
         jc	1f
         movzbw	%es:(%di), %bx
         cmpb	$0xff, %bl
-        je	2f
+        stc			/* initialise CY if jumping */
```

Comment:
Are we sure that 12xx clears carry?
Well I guess so, just confirming.

Reply:
It does. If it didn't this would have never worked because the `cmp` already sets CY if NZ. Pre-initialising CY for the ZR case only changes behaviour if the jump is taken.